src/commands/System/help.js

```diff
@@ -48,7 +48,7 @@ module.exports = class extends Command {
 				.then(() => {
 					if (!help.hasOwnProperty(command.category)) help[command.category] = {};
 					if (!help[command.category].hasOwnProperty(command.subCategory)) help[command.category][command.subCategory] = [];
-					help[command.category][command.subCategory].push(`${msg.guildSettings.prefix}${command.name.padEnd(longest)} :: ${command.description}`);
+					help[command.category][command.subCategory].push(`${msg.guildSettings.prefix.startsWith("//") ?  "\/\/" :msg.guildSettings.prefix}${command.name.padEnd(longest)} :: ${command.description}`);
```

Comment:
`msg.guildSettings.prefix.startsWith(“//“) ?  “\/\/“`

This breaks when the prefix is for example `//b`


` :msg.guildSettings.prefix}`
Missing a space before the `:`

It's also a pretty ugly fix and don't expect it to be merged. Instead transfer the command and fix it in your own bot.